docs/development/01-contribution.md

```diff
@@ -0,0 +1,45 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The active developers, which include the team that originally developed KILT, and the wider KILT community are engaged there, and can:
+
+- provide feedback on proposals or ideas
+- discuss possible use-cases or feature requests
+- make suggestions for non-technical contributions, including events, writing, or business models
+- answer questions about the protocol, services and products.
+
+## Feature Requests
+
+For the integrity of the protocol, requesting a feature requires approval from KILT governance. The request should follow the guidelines below, which are subject to change:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+
+ - Give your proposal a clear title
+ - Open an issue on the corresponding repository
+ - Provide a written outline of the feature request for discussion
+
+At the end of the discussion, the proposal needs the approval of the community, KILT Technical Committee, governance, or relevant project team (depending on the subject) to progress to the next stage. If approved, acceptable deliverables will be agreed with the proposer.
```

Comment:
So is this just for requesting features that KILT develops, or is it proposing a feature, that the proposer develops?

Reply:
I guess this could be both.

There could be a final closing remark here stating that the proposal or request must be taken by a team or done by the proposer.

Reply:
I like the split, but I think in the "Feature Proposal" should mention, that a proposal has to be made. because there is no proposal at that point.